### PR TITLE
Refactor to allow user to add address index

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
Fix issue #128

### CODE REVIEW

File: src/crypto/index.js [hunk 1]
```diff
@@ -266,17 +266,18 @@ export const validateMnemonic = bip39.validateMnemonic
 /**
  * Get a private key from mnemonic words.
  * @param {string} mnemonic the mnemonic phrase words
+ * @param {int} index the address index
  * @param {Boolean} derive derive a private key using the default HD path (default: true)
  * @return {string} hexstring
  */
-export const getPrivateKeyFromMnemonic = (mnemonic, derive = true) => {
+export const getPrivateKeyFromMnemonic = (mnemonic, index = 0, derive = true) => {
```
Comment (notatestuser) [inline src/crypto/index.js]: Would it be possible to move `index` to become the last arg so as to not break existing usages of this method?

Comment (bitcoinbrisbane) [reply src/crypto/index.js]: Good call.  Pushed.

**Comment by bitcoinbrisbane** [comment]
Also removed the `.only` in my test.

**Comment by notatestuser** [comment]
Thanks!

**Comment by bitcoinbrisbane** [comment]
Can close #41 now too I guess?

**Pull request merged**